micro_http/src/request.rs

```diff
@@ -178,11 +184,33 @@ impl<'a> Request<'a> {
     pub fn http_version(&self) -> Version {
         self.request_line.http_version
     }
+
+    /// Parses all the fields found in `bytes`, assuming the string starts right
+    /// after the request line.
+    pub fn parse_headers(&mut self, bytes: &[u8]) -> Result<bool, RequestError> {
```

Comment:
As per our offline conversation we decided to go against setting the body here. The body will be setted in a higher level function depending on the header "Content-Length" which would be parsed before.